Client/Frontend/Browser/TabManager.swift

```diff
@@ -900,7 +900,7 @@ extension TabManager: WKNavigationDelegate {
         tab.noImageMode = isNoImageMode
         
         if !tab.contentBlocker.isEnabled {
-            webView.evaluateJavaScript("window.__firefox__.TrackingProtectionStats.setEnabled(false, \(UserScriptManager.securityToken))", completionHandler: nil)
+            webView.evaluateSafeJavascript(functionName: "window.__firefox__.TrackingProtectionStats.setEnabled", args: [false, UserScriptManager.securityToken], sandboxed: false, completion: nil)
```

Comment:
`securityToken` is exposed via many helper functions to the web-page to avoid pages from over-writing the functions, so this new token will be used to protect web-pages from directly invoking the messageHandlers.